packages/cli/src/environments.ee/source-control/source-control-helper.ee.ts

```diff
@@ -160,9 +160,13 @@ export function getTrackingInformationFromPrePushResult(result: SourceControlled
 	};
 }
 
-export function getTrackingInformationFromPostPushResult(result: SourceControlledFile[]) {
+export function getTrackingInformationFromPostPushResult(
```

Comment:
I think the `source-control-helper.ee.test.ts` tests will need updating with the `userId` too.